Sources/MarigoldSwift/EngageBySailthru+Concurrency.swift

```diff
@@ -47,47 +46,6 @@ extension SailthruMobile {
         })
     }
     
-    
-    // MARK: Device
-    
-    /**
-     *  Asyncronously clears any of the Attribute, Message Stream, or Event data from the device.
-     *
-     *  Use this method to clear the device attributes after user logout.
-     *
-     *  - Parameter types: A bitwise OR collection of STMDeviceDataType dictating which sets of data to clear.
-     *  - Throws: Error when call fails.
-     **/
-    public func clearDeviceData(for types: STMDeviceDataType) async throws {
```

Comment:
We should replace this with a wrapper for the `clearAttributes` method that was added (also marked as deprecated though).